core/src/smartcontracts/wasm.rs

```diff
@@ -248,17 +320,16 @@ impl<'a, W: WorldTrait> Runtime<'a, W> {
     /// If decoding or execution of the query fails
     fn execute_query(
         mut caller: Caller<State<W>>,
+        query_id: u32,
```

Comment:
Does reducing this down to `u8` make a difference in size. Surely we don't have 256 query types.